client/src/index.jsx

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import Bonuses from './components/bonuses.jsx';
```

Comment:
We should confirm whether it is conventional to capitalize the names of component files if they are being imported with capital letters.

Reply:
Okay.